mp/src/materialsystem/stdshaders/pbr_dx9.cpp

```diff
@@ -250,10 +248,10 @@ END_SHADER_PARAMS
 
 			if (bHasMraoTexture)
 				BindTexture(SHADER_SAMPLER10, info.mraoTexture, 0);
-			else
-				pShaderAPI->BindStandardTexture(SHADER_SAMPLER10, TEXTURE_WHITE);
+	  else
```

Comment:
Would you mind using tab indents here as well?

Reply:
don